specs/kiota.config.md

```diff
@@ -0,0 +1,634 @@
+```
+
+### Remove an API
+
+```bash
+kiota api delete --name "My Api" --clean-output
+```
+
+## JSON Schema for Kiota.Config
+
+```json
+{
+  "$schema": "<http://json-schema.org/draft-07/schema#>",
+  "type": "object",
+  "properties": {
+    "name": {
+      "type": "string"
+    },
+    "apis": {
+      "type": "object",
+      "patternProperties": {
+        ".*": {
+          "type": "object",
+          "properties": {
+            "descriptionLocation": {
+              "type": "string"
+            },
+            "descriptionHash": {
+              "type": "string"
+            }
+          },
+          "descriptionHash": {
+            "type": "string"
+          },
+          "descriptionLocation": {
+            "type": "string"
+          },
+          "includePatterns": {
+            "type": "array",
+            "items": {
+              "type": "string"
+            }
+          },
+          "excludePatterns": {
+            "type": "array",
+            "items": {
+              "type": "string"
+            }
+          },
+          "baseUrl": {
+            "type": "string"
+          },
+          "clients": {
+            "type": "object",
+            "patternProperties": {
+              ".*": {
+                "type": "object",
+                "properties": {
+                  "language": {
+                    "type": "string"
+                  },
+                  "outputPath": {
+                    "type": "string"
+                  },
+                  "clientClassName": {
+                    "type": "string"
+                  },
+                  "clientNamespaceName": {
+                    "type": "string"
+                  },
+                  "features": {
+                    "type": "object",
+                    "properties": {
+                      "structuredMediaTypes": {
+                        "type": "array",
+                        "items": {
+                          "type": "string"
+                        }
+                      },
+                      "serializers": {
+                        "type": "array",
+                        "items": {
+                          "type": "string"
+                        }
+                      },
+                      "deserializers": {
+                        "type": "array",
+                        "items": {
+                          "type": "string"
+                        }
+                      },
+                      "usesBackingStore": {
+                        "type": "boolean"
+                      },
+                      "includeAdditionalData": {
+                        "type": "boolean"
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          }
+        },
+        "disabledValidationRules": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    }
+  }
+}
```

Comment:
We might be missing commands to add new paths/operations to existing clients. Or maybe the add command already does that for you but it's not called out?
We might be missing a command to remove a path/operation from a client, without deleting the client itself?